docs/FAQ.md

```diff
@@ -137,6 +137,17 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I resume working on an existing change?
+
+There are two ways to resume working on an earlier change: `jj new` then `jj squash`,
+and `jj edit`. The first is generally recommended over the second because at this
+point, your earlier change could have developed a conflict. In that case`jj edit`
```

Comment:
Does this mean that we recommend `jj new` only there is a conflict?

FWIW, I think if it more like this:

> To resume working on an unfinished change, use `jj edit`. If the change is pretty much done, use `jj new` instead to be able to easily see your modifications with `jj diff` before squashing them in. If a commit has children, then that's a clear sign that "the change is pretty much done". Also, if the commit has conflicts, use `jj new` to keep the conflict resolution separate.

But I know that that's not how everyone thinks of it, so make sure you get other people's input on this PR. Maybe ask on Discord if you need more input.

Reply:
Capturing my summary from Discord
> We should recommend new+squash as the default, highlight the two conditions that can make edit less ideal, but make sure edit isn't made out to be less overall for the possible pitfalls.

Reply:
I've pushed a rewrite